### PR TITLE
style: update band logos

### PR DESCRIPTION
**Pull request by AnaisUrlichs**

### CODE REVIEW

**Comment by simar7** [comment]
Are new t-shirts part of the code review as well? 🆕

File: docs/imgs/logo.png
Comment (knqyf263) [inline docs/imgs/logo.png:1]: I think the logo should be smaller to reduce the image size.

**Review by knqyf263 — COMMENTED** [review]
The PR title must comply with the policy.
https://aquasecurity.github.io/trivy/v0.48/community/contribute/pr/#title

File: docs/imgs/logo.png
Comment (AnaisUrlichs) [reply docs/imgs/logo.png:1]: ok, will update it

**Review by itaysk — APPROVED** [review]
lgtm. one nit is that file names still say "2022".

**Comment by knqyf263** [comment]
@itaysk Do we want to keep the year in the file names? Can we remove the year like `brand/Trivy-OSS-Logo-Color-Horizontal-RGB.png`?

**Comment by AnaisUrlichs** [comment]
@knqyf263 @itaysk I wasn't sure if the file names are used anywhere -- I meant to ask you as I could not find them in the trivy repo being used -- otherwise, I would just remove the year name?

**Comment by itaysk** [comment]
yes removing the year sounds good

**Pull request merged**